web/src/user_groups.ts

```diff
@@ -433,8 +434,12 @@ export function get_associated_subgroups(user_group: UserGroup, user_id: number)
     return subgroups;
 }
 
-export function group_list_to_comma_seperated_name(user_groups: UserGroup[]): string {
-    return user_groups.map((user_group) => user_group.name).join(", ");
+export function format_group_list(user_groups: UserGroup[]): string {
+    return util.format_array_as_list(
+        user_groups.map((user_group) => user_group.name),
+        "short",
+        "conjunction",
+    );
```

Comment:
I still kept the wrapper function around so that if we want to ever change `short` -> `long`, it's easier to do